src/chronometer.js

```diff
@@ -1,33 +1,47 @@
 class Chronometer {
   constructor() {
-    // ... your code goes here
+    this.intervalId = null; 
+    this.currentTime = 0;
   }
-
+  
   start(printTimeCallback) {
-    // ... your code goes here
-  }
+      this.intervalId = setInterval(() => {
+      this.currentTime++;
+      if (printTimeCallback){
+        printTimeCallback();
+      }
+    }, 1000);
+ 
+}
 
   getMinutes() {
-    // ... your code goes here
+    return Math.floor(this.currentTime / 60);
   }
 
   getSeconds() {
-    // ... your code goes here
-  }
+    return (this.currentTime % 60);
+    }
 
   computeTwoDigitNumber(value) {
-    // ... your code goes here
+    this.value = value;
+    let twoDigitNumber = this.value.toFixed(0);
+    if (twoDigitNumber.length === 1){
+      return '0' + twoDigitNumber;
+    } 
+    return twoDigitNumber;
   }
 
   stop() {
-    // ... your code goes here
+    clearInterval(this.intervalId);
   }
 
   reset() {
-    // ... your code goes here
+    this.currentTime = 0;
   }
 
   split() {
-    // ... your code goes here
+    let twoDigitMinute = (this.computeTwoDigitNumber(this.getMinutes()))
+    let twoDigitSecond = (this.computeTwoDigitNumber(this.getSeconds()))
+     return `${twoDigitMinute}` + `:` + `${twoDigitSecond}`
```

Comment:
I have `+ '.' + `${twoDigitCentisecond}` at the end of the return here, but for whatever reason it doesn't show up here.

Reply:
update: it can now be seen in my more recent commit